.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "terraform-provider-coder",
+  "image": "mcr.microsoft.com/devcontainers/go:1.20",
```

Comment:
Do we still use Go 1.20 here, btw? Should we match coder/coder or go 1.22? 🤔

Reply:
https://github.com/coder/terraform-provider-coder/issues/222